src/firefly/java/edu/caltech/ipac/table/io/FITSTableReader.java

```diff
@@ -393,9 +540,94 @@ private static DataGroup convertStarTableToDataGroup(StarTable table,
         }
         dataGroup.trimToSize();
         return dataGroup;
+    }*/
+
+    static class EvalVal {
```

Comment:
You can make this a `record`. It will take half the space.

Reply:
Ah yes, you mentioned it above too, my bad. Changed to a record and pushed code into existing commit.